yapf/yapflib/reformatter.py

```diff
@@ -277,7 +278,8 @@ def _AlignTrailingComments(final_lines):
     for tok in line.tokens:
       if (tok.is_comment and isinstance(tok.spaces_required_before, list) and
           tok.value.startswith('#')):
-        # All trailing comments and comments that appear on a line by themselves
+        # All trailing comments
+        # NOTE not including comments that appear on a line by themselves
```

Comment:
Please reflow this comment. And place any "NOTE" comments at the end of the comment block.